.vscode/settings.json

```diff
@@ -83,7 +83,9 @@
   "markdownlint.config": {
     "MD024": {
       "allow_different_nesting": true
-    }
+    },
+    "MD033": false,
```

Comment:
fyi for those interested

https://github.com/DavidAnson/markdownlint/blob/main/doc/md033.md
https://github.com/DavidAnson/markdownlint/blob/main/doc/md041.md